docs_app/content/guide/operators.md

```diff
@@ -87,256 +110,190 @@ const fileObservable = urlObservable.pipe(
+  Observable.
+
+Just as many array libraries combine {@link
+https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map map} and {@link
+https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flat flat} (or `flatten()`) into
+a single {@link https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flatMap flatMap},
+there are mapping equivalents of all the RxJS flattening operators {@link concatMap}, {@link mergeMap},
+{@link switchMap}, and {@link exhaustMap}.
 
 ## Marble diagrams
 
-To explain how operators work, textual descriptions are often not enough. Many operators are related to time, they may for instance delay, sample, throttle, or debounce value emissions in different ways. Diagrams are often a better tool for that. _Marble Diagrams_ are visual representations of how operators work, and include the input Observable(s), the operator and its parameters, and the output Observable.
+To explain how operators work, textual descriptions are often not enough. Many operators are related to time, they may
+for instance delay, sample, throttle, or debounce value emissions in different ways. Diagrams are often a better tool
+for that. _Marble Diagrams_ are visual representations of how operators work, and include the input Observable(s), the
+operator and its parameters, and the output Observable.
 
-<span class="informal">In a marble diagram, time flows to the right, and the diagram describes how values ("marbles") are emitted on the Observable execution.</span>
+<span class="informal">In a marble diagram, time flows to the right, and the diagram describes how values ("marbles")
+are emitted on the Observable execution.</span>
 
 Below you can see the anatomy of a marble diagram.
 
 <img src="assets/images/guide/marble-diagram-anatomy.svg">
 
-Throughout this documentation site, we extensively use marble diagrams to explain how operators work. They may be really useful in other contexts too, like on a whiteboard or even in our unit tests (as ASCII diagrams).
+Throughout this documentation site, we extensively use marble diagrams to explain how operators work. They may be really
+useful in other contexts too, like on a whiteboard or even in our unit tests (as ASCII diagrams).
 
 ## Categories of operators
 
-There are operators for different purposes, and they may be categorized as: creation, transformation, filtering, joining, multicasting, error handling, utility, etc. In the following list you will find all the operators organized in categories.
-
-For a complete overview, see the [references page](/api).
-
-### <a id="creation-operators-list"></a>Creation Operators
-
-- [`ajax`](/api/ajax/ajax)
-- [`bindCallback`](/api/index/function/bindCallback)
-- [`bindNodeCallback`](/api/index/function/bindNodeCallback)
-- [`defer`](/api/index/function/defer)
-- [`EMPTY`](/api/index/const/EMPTY)
```

Comment:
`EMPTY` is not a creation operator, `empty()` was one - thus, I removed `EMPTY` entirely.

Reply:
No worries, I didn't realize it as well until I had to rebase master.